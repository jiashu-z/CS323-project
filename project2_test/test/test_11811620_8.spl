struct A
{
  int num;
};
int func3(int a3[9],int b3[9][9], struct A c3[9], struct A d3[9][9]){
  return a3[0];
}
int test_11811620_4()
{
  int b[10][10];
  struct A c[10][10];
  func3(b[0], b, c[0], c);
  func3(b, b, c, c);
  return 0;
}