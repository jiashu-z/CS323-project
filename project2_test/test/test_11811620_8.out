Error type 9 at Line 13: invalid argument type for func3. expected:array (dimension:1), actual:array (dimension:2)